src/App.tsx

```diff
@@ -14,143 +57,27 @@ export const App: React.FC = () => {
       <h1 className="todoapp__title">todos</h1>
 
       <div className="todoapp__content">
-        <header className="todoapp__header">
-          {/* this buttons is active only if there are some active todos */}
-          <button type="button" className="todoapp__toggle-all active" />
-
-          {/* Add a todo on form submit */}
-          <form>
-            <input
-              type="text"
-              className="todoapp__new-todo"
-              placeholder="What needs to be done?"
+        <Header
+          removeError={() => setError(false)}
```

Comment:
Why do you need this method in the Header? The error should be shown in the Footer as it is shown in the working version.

Reply:
![Uploading image.png…]()